kasa/cli.py

```diff
@@ -696,6 +774,11 @@ async def state(ctx, dev: Device):
         for module in dev.modules.values():
             echo(f"\t[green]+ {module}[/green]")
 
+        for child in dev.children:
+            echo(f"\n\t[bold]== Child {child.alias} Modules ==[/bold]")
+            for module in child.modules.values():
+                echo(f"\t\t[green]+ {module}[/green]")
+
```

Comment:
This was helpful while testing the extra modules on the `iot` smartstrips but can be removed and not a huge part of this PR.

Reply:
That's done.  I moved the verbose module output up before the children as well as it looked odd coming after.